catalogo.html

```diff
@@ -17,64 +16,148 @@
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#"
+                  >Catálogo</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="cotiza.html">Cotizá tu auto</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="contacto.html">Contacto</a>
+              </li>
+            </ul>
           </div>
         </div>
       </nav>
     </header>
 
-    <main>
-      <section>
-        <h2 id="autos">Nuestros autos</h2>
-        <div class="autos-container">
-          <!-- Auto 1 -->
-          <article class="auto-card">
-            <img src="assets/onix.avif" alt="Auto Chevrolet" />
-            <p>Nombre auto - Modelo - Kilómetros</p>
-            <p><strong>$Precio</strong></p>
-            <button>Me interesa</button>
-          </article>
+    <main class="container-catalogo my-4">
+      <div class="row">
+        <!-- Aside de filtros -->
+        <aside class="aside-catalogo col-md-3">
+          <form id="form-filtros" onsubmit="filtrarAutos(event)">
+            <div class="mb-3">
+              <h2>Filtros</h2>
+              <label for="marca" class="form-label">Marca</label>
+              <select class="form-select" id="marca">
+                <option value="">Todos</option>
+                <option>Chevrolet</option>
+                <option>Renault</option>
+                <option>Toyota</option>
+              </select>
+            </div>
 
-          <!-- Auto 2 -->
-          <article class="auto-card">
-            <img src="assets/sandero.jpg" alt="Auto Renault" />
-            <p>Nombre auto - Modelo - Kilómetros</p>
-            <p><strong>$Precio</strong></p>
-            <button>Me interesa</button>
-          </article>
+            <div class="mb-3">
+              <label for="precioMax" class="form-label">Precio máximo</label>
+              <input
+                type="number"
+                class="form-control"
+                id="precioMax"
+                placeholder="$"
+              />
+            </div>
 
-          <!-- Auto 3 -->
-          <article class="auto-card">
-            <img src="assets/yaris.png" alt="Auto Toyota" />
-            <p>Nombre auto - Modelo - Kilómetros</p>
-            <p><strong>$Precio</strong></p>
-            <button>Me interesa</button>
-          </article>
-        </div>
-      </section>
+            <div class="mb-3">
+              <label for="kmMax" class="form-label">Máximo de kilómetros</label>
+              <input
+                type="number"
+                class="form-control"
+                id="kmMax"
+                placeholder="e.g. 50000"
+              />
+            </div>
+
+            <button type="submit">Aplicar filtros</button>
+          </form>
+
+          <form id="form-agregar-auto" onsubmit="agregarAuto(event)">
```

Comment:
Eso está mal desde una buena práctica moderna de desarrollo web. Usar manejadores de eventos en línea (inline), no es la forma recomendada de manejar eventos en JavaScript. Si tenés muchos eventos definidos en línea, el HTML se vuelve difícil de mantener.
Esto rompe el principio de separación de responsabilidades (SoC - Separation of Concerns): el HTML debería encargarse de la estructura y contenido, mientras que el JS debería manejar la lógica y comportamiento.
```suggestion
document.getElementById('form-agregar-auto').addEventListener('submit', agregarAuto);
```